cpp/src/io/parquet/reader_impl_helpers.cpp

```diff
@@ -64,11 +83,11 @@ thrust::optional<LogicalType> converted_to_logical_type(SchemaElement const& sch
  */
 type_id to_type_id(SchemaElement const& schema,
                    bool strings_to_categorical,
-                   type_id timestamp_type_id)
+                   type_id timestamp_type_id,
+                   cudf::data_type duration_type)
```

Comment:
Do we need the `data_type` or just the `type_id`?

Reply:
either or works here.

Reply:
No longer in use.